anabot/runtime/installation/hub/software_selection.py

```diff
@@ -216,3 +223,43 @@ def addon_check(element, app_node, local_node):
     if action_result(element)[0] == False:
         return action_result(element)
     return addon_handler_manipulate(element, app_node, local_node, True)
+@handle_act('/kernel_options')
+def kernel_options_handler(element, app_node, local_node):
+    """ Functions correctly only in english as I didn't find a way to translate the full kernel options."""
```

Comment:
I may be wrong as I haven't dug very deep into this, but think I might know what the issue is. After having a look at your code and Anaconda messages and their translations, it seems that you're trying to translate the full string from the combo box, however it's actually composed from two separate translatable strings separated by `\n`. You'd have to translate them separately and then it will likely work (other than that, there might be some HTML-like formatting included). Running `app_node.dump()` via `/opt/debug_dogtail.py` in Anaconda environment with Anabot updates image might give you a clue how the combo box string you're looking for looks like. The only strange thing is that you can find the combo box only by the `DEFAULT_KERNEL_OPTION` string (lines 233 and 262).

You can also have a look at the messages and their translations, e. g. for Czech (run in installation environment):
`msgunfmt /usr/share/locale/cs/LC_MESSAGES/anaconda.mo | grep -Ea1 "More efficient|System performance|6?4k"

You can give it a try to achieve a successful translation; however if you find there's another unexpected obstacle, it wouldn't probably be worth spending much more time on this.

Reply:
I looked into it and the split of the message into two separate ones was half of the success, the other half was the removal of the context, that made it to work incorrectly.

So thank you once again for your help, the translations now work correctly :)